src/app/tests/suites/certification/Test_TC_DGGEN_2_1.yaml

```diff
@@ -258,7 +269,9 @@ tests:
     - label: "Step 5a: Reboot target device(DUT)"
       PICS: PICS_USER_PROMPT
       verification: |
-          Did the DUT successfully reboot?
+          execute the below mentioned command to put DUT into a commissionable state, Pls use equivalent command on the respective DUT
```

Comment:
All these comments are wrong; see above.

Reply:
Reverted back to the previous version.